packages/expo-constants/src/Constants.types.ts

```diff
@@ -111,7 +117,23 @@ export interface AppManifest extends ExpoConfig {
    * between accounts or renamed.
    */
   projectId?: string;
+}
 
+/**
+ * A classic manifest https://docs.expo.io/guides/how-expo-works/#expo-manifest
+ */
+export interface AppManifest extends ExpoClientConfig, ExpoGoConfig {
+  // this needs to be specified to appease typescript
```

Comment:
Instead of `interface`, would `type` solve this? Years ago there was a post online comparing `type` to `interface` that talked about some of the limitations of `type`. Fewer years ago, TypeScript changed and those limitations went away. Now I believe they are almost equally powerful except that `interface` lets classes implement them. Since we don't use any classes here, `type` seems appropriate.